### DISCUSSION TITLE
Add Portugues language

### DISCUSSION THREAD

**Discussion by phcrepaldi** [RajSolai/TextSnatcher#29] [Ideas]
This app is fantastic, I use Linus Mint and I wanted to add Portugues language but unfortunately I'm not able to recompile the code because one of the libraries dependencies does not run on Mint for it is based on Ubuntu 18.04 (libhandy-1.0). Is there any way to add Portuguese language? Wonderful job. Congratulations.

**Comment by RajSolai** [comment]
Thanks for your support @phcrepaldi, I will add Portuguese language support by next release of TextSnatcher.
